catkin_ws/src/state_estimation/src/simple_state_estimation.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+import rospy
+import math
+import numpy as np
+
+from geometry_msgs.msg import Point, Vector3, Quaternion
+from sensor_msgs.msg import Imu, FluidPressure
+
+from aquadrone_msgs.msg import SubState
+
+class StateEstimator:
+
+    def __init__(self):
+        # World Frame
+        self.position = Point()
+        self.pos_variance = Vector3(*(1, 1, 1))
+
+        # Local Frame
+        self.velocity = Vector3(*(0, 0, 0))
+        self.vel_variance = Vector3(*(1, 1, 1))
+
+        # World Frame
+        self.orientation = Quaternion()
+    
+        self.imu_sub = None
+        self.depth_sub = None
+        self.state_pub = None
+        self.state_msg = SubState()
+
+        self.pressure_offset = 100.0
+        self.g = 9.8
+
+        self.rate = 10
+        self.rate_ctrl = rospy.Rate(self.rate)
+
+    def increment_dynamics(self):
+        dp = self.velocity.x * 1.0/self.rate
+        self.position.x += dp
+        self.pos_variance.x +=  0.01 * dp
+
+        dp = self.velocity.y * 1.0/self.rate
+        self.position.y += dp
+        self.pos_variance.y +=  0.01 * dp
+
+        dp = self.velocity.z * 1.0/self.rate
+        #self.position.z += dp
+        #self.pos_variance.z +=  0.01 * dp
+
+    def imu_cb(self, msg):
```

Comment:
cb stands for calibration?

Reply:
callback. I'll look into making it more clear.